src/viewer.js

```diff
@@ -434,7 +434,7 @@ export class Viewer {
    * @param {boolean} awaitResponse
    * @return {(!Promise<*>|undefined)}
    * @package
-   * @export
+   * _export
```

Comment:
There is no magic here anyway. For a long time we won't be able to run with property obfuscation anyway.